README.Rmd

```diff
@@ -29,10 +29,6 @@ The pins package publishes data, models, and other R objects, making it easy to
 You can pin objects to a variety of pin *boards*, including folders (to share on a networked drive or with services like DropBox), Posit Connect, Amazon S3, Google Cloud Storage, Azure storage, and Microsoft 365 (OneDrive and SharePoint).
 Pins can be automatically versioned, making it straightforward to track changes, re-run analyses on historical data, and undo mistakes.
 
-pins 1.0.0 includes a new more explicit API and greater support for versioning.
```

Comment:
pins 1.0.0 was in Oct 2021 so I think it's time to remove this, to avoid confusion for folks coming to this page afresh. The vignette about moving to pins 1.0.0 is still included.